### PR TITLE
feat: `v8.0.0-rc.4` upgrade handler

### PR DESCRIPTION
**Pull request by boojamya**
This PR modify's the upgrade handler to add the `globalfee` module to testnet.

It also adds back a modified and simplified version of the upgrade test that was present in older versions of Noble.
I envision this test being ported over to the main the branch when it becomes needed.

### CODE REVIEW

File: e2e/upgrade_utils.go [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zaptest"
+)
+
+const (
+	haltHeightDelta    = 10 // will propose upgrade this many blocks in the future
+	blocksAfterUpgrade = 10
+
+	authModule = "noble13am065qmk680w86wya4u9refhnssqwcvgs0sfk"
```
Comment (johnletey) [inline e2e/upgrade_utils.go:38]: Quick heads up that you can get this from the Authority module types directly!

**Review by johnletey — APPROVED** [review]
Incredible job here! @boojamya

File: e2e/cctp_roles_test.go [hunk 2]
```diff
@@ -34,7 +34,7 @@ func TestCCTP_UpdateOwner(t *testing.T) {
 	t.Parallel()
 
 	ctx := context.Background()
-	nw := e2e.NobleSpinUp(t, ctx, true)
+	nw, _ := e2e.NobleSpinUp(t, ctx, e2e.LocalImages, true)
```
[the remark this answers is not in the document]
Comment (boojamya) [reply e2e/cctp_roles_test.go:37]: Doesn't return error

File: e2e/upgrade_utils.go (hunk 1 above)
Comment (boojamya) [reply e2e/upgrade_utils.go]: Done! [`fc4f835` (#422)](https://github.com/noble-assets/noble/pull/422/commits/fc4f8357a339d4231913f026d628ecf27db7a82c)

**Review by johnletey — APPROVED** [review]

**Pull request merged**